spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -150,6 +150,10 @@ timeToLive::
 See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html[DynamoDB TTL]
 +
 No default - means no records expiration.
+interval::
+  The interval, in milliseconds, between two checkpoints when checkpoint mode is `periodic`.
++
+No default - if not set, default value inside spring-integration-aws will be used (5000).
```

Comment:
I don't think we can say "No default". There is indeed that `5 seconds` default and as you see by all other `KinesisConsumerProperties` we provide that default value there. 
So, please, after moving this `interval` to the `KinesisConsumerProperties`, add also such a default value.